go.mod

```diff
@@ -66,22 +59,26 @@ require (
 	github.com/zeebo/errs v1.2.2
 	go.uber.org/atomic v1.4.0
 	go.uber.org/goleak v0.10.0
-	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
-	golang.org/x/net v0.0.0-20190620200207-3b0461eec859
+	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
+	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
 	golang.org/x/sys v0.0.0-20200122134326-e047566fdf82
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
 	golang.org/x/tools v0.0.0-20191127201027-ecd32218bd7f
 	google.golang.org/api v0.6.0
 	google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
 	google.golang.org/grpc v1.27.1
 	gopkg.in/DATA-DOG/go-sqlmock.v1 v1.3.0 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/square/go-jose.v2 v2.4.1
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.0.0-20190222213804-5cb15d344471
-	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
+	k8s.io/api v0.18.2
+	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v10.0.0+incompatible
-	k8s.io/klog v1.0.0 // indirect
-	sigs.k8s.io/yaml v1.1.0 // indirect
+	sigs.k8s.io/controller-runtime v0.6.0
 )
+
+replace k8s.io/client-go => k8s.io/client-go v0.18.2
+
+replace k8s.io/api => k8s.io/api v0.18.2
+
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.18.2
```

Comment:
Something weird with the go modules. Those three have to be in sync. Without the replace's one of them ends up using an older version

Reply:
I think it's because you haven't changed `k8s.io/client-go` to `v0.18.2`? It's still set to `v10.0.0+incompatible`. In any case, I spent some time trying to update these dependencies to discover what you were hitting (before I noticed `client-go` not getting bumped) and uncovered an update that is required in some of our code that interacts with `golang.org/x/crypto` which is also being bumped when moving to v0.18.2. If you are ok with it, I'll open up a separate PR that bumps this dependency while fixing up our code that interacts with golang.org/x/crypto.